### PR TITLE
Update installing-hiyacfw.md

### PR DESCRIPTION
**Pull request by wbrtm**
Add step of installing tk package under linux

### CODE REVIEW

**Comment by urmum-69** [comment]
This doesn't apply to all distributions - this python3-tk package is for debian and ubuntu specifically, which I wouldn't assume people are using

**Comment by Emberwit** [comment]
Thanks, I changed the bullet point to not include a specific package name.

**Comment by Epicpkmn11** [comment]
Would saying "Run `pip3 -r requirements.txt`" or so be better? Or is pip not commonly included on linux that leaving it vague is best?

**Comment by urmum-69** [comment]
Pip is usually included, although sometimes might require installing a package like `python-pip`
I think that would be better in general though

**Comment by Emberwit** [comment]
Made another change using your feedback.

**Comment by NightYoshi370** [comment]
Maybe try including "If your system does not have pip installed, install the `python-pip` package" as a sub-bullet

File: _pages/en_US/installing-hiyacfw.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ This section is entirely optional. If Unlaunch is enough for you, you can stop h
 1. Install Python 3 using your package manager if its not already installed
 1. Download the latest Python version of [hiyaCFW Helper](https://github.com/mondul/HiyaCFW-Helper/releases)
 1. Extract the hiyaCFW Helper archive anywhere on your PC
+1. Make sure the neccessary Python packages are installed running `pip3 install -r requirements.txt` in the directory containing the extracted files
```
Comment (Epicpkmn11) [inline _pages/en_US/installing-hiyacfw.md:60]:
```suggestion
1. Make sure the necessary Python packages are installed by running `pip3 install -r requirements.txt` in the directory containing the extracted files
   - You can install pip by running `python3 -m ensurepip`
```
```suggestion
1. Make sure the necessary Python packages are installed by running `pip3 install -r requirements.txt` in the directory containing the extracted files
   - You can install pip with the `python3-pip` package
```
Here's a couple suggestions of a couple minor typo fixes and for installing pip, I think the first is more universally compatible, but I'm not sure if that module is always included

**Pull request merged**